README.md

```diff
@@ -46,21 +46,21 @@
-|  | | [DIDComm](https://github.com/decentralized-identity/didcomm-messaging) |  [ACDC](https://wiki.trustoverip.org/display/HOME/ACDC+%28Authentic+Chained+Data+Container%29+Task+Force) |  [Credential Issuance Testing Tool](https://identity-wallet-credential-issuance-web-interface.vercel.app/)  |  [Signify-TS](https://github.com/cardano-foundation/signify-ts) | 
-|  | | [KERI on Cardano](https://github.com/cardano-foundation/cardano-backer) | |  | |
+| **User Interface**    |  **Secure Storage**  |  **Standards & Protocols** |  **SSI Services**   |           
+|----------------|---------------|---------------|----------------|
+|  Intuitive User Experience |  Hardware Security Module (HSM)   | [KERI on Cardano](https://github.com/cardano-foundation/cardano-backer) | [KERIA](https://github.com/cardano-foundation/keria) | 
+|  Android  |  Secure Enclave (SE) | [Autonomic Identifier](https://weboftrust.github.io/WOT-terms/docs/glossary/autonomic-identifier) |  [Signify-TS](https://github.com/cardano-foundation/signify-ts) | 
+| iOS | Private Key Storage | [ACDC](https://wiki.trustoverip.org/display/HOME/ACDC+%28Authentic+Chained+Data+Container%29+Task+Force) |  [Credential Issuance Modules](https://identity-wallet-credential-issuance-web-interface.vercel.app/)  |   | 
+
 
 # Potential Future Developments
 
-|**App Features**            |  **Identity Recovery**               |     **Identifiers**           |     **Verifiable Credentials**  |    **Communications** |   **Peer-to-Peer (P2P)** |  
-|----------------|---------------|---------------|----------------|----------------|----------------|
-| Bluetooth |  [Shamir Secret Sharing](https://en.wikipedia.org/wiki/Shamir%27s_secret_sharing)|  [did:webs](https://labs.hyperledger.org/labs/didwebs.html#:~:text=did%3Awebs%20extends%20the%20web,%2C%20X509%2C%20and%20certificate%20authorities.) |  [OID4vc](https://openid.net/sg/openid4vc/specifications/)| Messaging | [CIP-45](https://github.com/cardano-foundation/CIPs/pull/395) Support  | 
-| Biometrics | On-chain Recovery | [did:keri](https://identity.foundation/keri/did_methods/)  | On-Chain Schemas | Social Recovery |  App Integration |
-| NFC|  Local Encrypted|  | [AnonCreds](https://www.hyperledger.org/projects/anoncreds) | [DIDComm v2](https://identity.foundation/didcomm-messaging/spec/) | | 
-|Aries Askar | Cloud-based |  | [Cheqd](https://cheqd.io/) | |  |
+|**Mobile Application Features**        |  **Decentralized Identifier Recovery**               |     **Standards & Protocols**  |  **Peer-to-Peer (P2P)** |  
+|----------------|---------------|----------------|----------------|
+| Bluetooth |  [Shamir Secret Sharing](https://en.wikipedia.org/wiki/Shamir%27s_secret_sharing)| Ledger-based Schemas  | Private Messaging | 
+| Biometrics | On-chain Recovery | Direct Mode (Offline) | Social Recovery |  App Integration |
```

Comment:
If `Direct Mode` here is direct mode KERI it seems out of place (and technically we have this already as a config)